### PR TITLE
Translate "algorithm" into Ukrainian

### PR DESCRIPTION
**Pull request by olexandr-konovalov**
## Author: 

- Olexandr Konovalov

## Language: 

- Ukrainian

## Terms defined:

- algorithm

### CODE REVIEW

**Review by elletjies — APPROVED** [review]

File: glossary.yml [hunk 1]
```diff
@@ -776,12 +776,16 @@
     def: >
       Um algoritmo é um conjunto de etapas, instruções ou regras a serem seguidas para realizar uma tarefa específica.
       Na ciência da computação, um algoritmo é um conjunto de instruções em um programa de computador que resolve um problema computacional.
+  uk:
+    term: "алгоритм"
+    def: >
+      Алгоритм — це набір кроків, інструкцій або правил, яких слід дотримуватися для виконання конкретного завдання.
```
Comment (olexandr-konovalov) [inline glossary.yml:782]: Need to add "впорядкований"

Comment (elletjies) [reply glossary.yml:782]: @olexandr-konovalov great - can you make the changes for us?

Comment (olexandr-konovalov) [reply glossary.yml:782]: @elletjies yes, I will! I was using this for a demo of how to make comments on GitHub to a colleague who will be involved in Ukrainian Glosario. I will now use this to show how to make a PR next week :)

Comment (olexandr-konovalov) [reply glossary.yml:782]: PR #657 adds this

**Pull request merged**